### PR TITLE
Feature post requests

### PR DESCRIPTION
**Pull request by HannesWaechter**
create folder, create album and upload file methods for canto client

### CODE REVIEW

File: src/canto/client.py [hunk 1]
```diff
@@ -106,15 +111,28 @@ def refresh_access_token(self, refresh_token):
...
+            raise ApiException(
+                'method should be one of "get" or "post". Got "{}".'.format(method)
+            )
+
+        if method == 'post':
+            request_method = requests.post
+        else:
+            request_method = requests.get
+
+        response = request_method(
```
Comment (RaphaelKimmig) [inline src/canto/client.py]: `requests.request(request_method, ...)`

File: src/canto/client.py [hunk 2]
```diff
@@ -219,3 +240,69 @@ def get_tree(self):
...
+        data["acl"] = setting["acl"]
+        data["AWSAccessKeyId"] = setting["AWSAccessKeyId"]
+        data["Policy"] = setting["Policy"]
+        data["Signature"] = setting["Signature"]
+
+        data["x-amz-meta-file_name"] = "${filename}"
+        data["x-amz-meta-tag"] = ""
+        data["x-amz-meta-scheme"] = scheme  # set original scheme if updating
+        data["x-amz-meta-id"] = image_id  # set original id if updating
+        data["x-amz-meta-album_id"] = album_id
```
Comment (RaphaelKimmig) [inline src/canto/client.py]: a regular dict would be just fine :-)

File: src/canto/client.py [hunk 3]
```diff
@@ -24,6 +25,10 @@ class ResourceNotFound(ApiException):
     pass
 
 
+class AccessTokenRequired(ApiException):
```
Comment (RaphaelKimmig) [inline src/canto/client.py:28]: not sure if this should be api exception but don't really know what it should be, so whatever

**Review by RaphaelKimmig — CHANGES_REQUESTED** [review]

File: src/canto/client.py (hunk 2 above)
Comment (HannesWaechter) [reply src/canto/client.py]: ok

File: src/canto/client.py (hunk 3 above)
Comment (HannesWaechter) [reply src/canto/client.py:28]: ok

**Pull request merged**